lib/ansible/modules/network/ios/ios_lldp_global.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for ios_lldp_global
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+
+DOCUMENTATION = """
+---
+module: ios_lldp_global
+version_added: 2.9
+short_description: Configure and manage Link Layer Discovery Protocol(LLDP) attributes on IOS platforms.
+description: This module configures and manages the Link Layer Discovery Protocol(LLDP) attributes on IOS platforms.
+author: Sumit Jaiswal (@justjais)
+notes:
+  - Tested against Cisco IOSv Version 15.2 on VIRL
+  - This module works with connection C(network_cli),
+    See L(IOS Platform Options,../network/user_guide/platform_ios.html).
+options:
+  config:
+    description: A dictionary of LLDP options
+    type: dict
+    suboptions:
+      holdtime:
+        description:
+          - LLDP holdtime (in sec) to be sent in packets.
+          - Refer to vendor documentation for valid values.
+        type: int
+      reinit:
+        description:
+          - Specify the delay (in secs) for LLDP to initialize.
+          - Refer to vendor documentation for valid values.
+          - NOTE, if LLDP reinit is configured with starting value idempotency
+            won't be maintained as Cisco device doesn't record starting reinit
+            configured value, so it cannot be verified that if the respective
+            starting reinit value is already configured or not from the device
+            side. So, if user uses tries to apply starting reinit value in every
+            play run ansible will show changed as True. But, for other reinit value
+            idempotency will be maintained as other than starting reinit value other
+            reinit value are recorded in device side.
+        type: int
+      run:
```

Comment:
iirc other platform use `enable` option to enable/disable lldp

Reply:
updated to `enable`